src/lib.rs

```diff
@@ -1323,6 +1387,35 @@ pub struct AudioInputPacket {
     implementation: *mut IDeckLinkAudioInputPacket,
 }
 
+impl AudioInputPacket {
+    pub fn get_sample_frame_count(&mut self) -> usize {
+        unsafe { decklink_audio_input_packet_get_sample_frame_count(self.implementation) as _ }
+    }
+
+    pub fn get_bytes_raw(&mut self) -> Result<*const u8, Error> {
+        unsafe {
+            let mut buf: *mut c_void = std::ptr::null_mut();
+            void_result(decklink_audio_input_packet_get_bytes(
+                self.implementation,
+                &mut buf,
+            ))?;
+            Ok(buf as *mut u8)
+        }
```

Comment:
```suggestion
        let mut buf: *mut c_void = std::ptr::null_mut();
        unsafe {
            void_result(decklink_audio_input_packet_get_bytes(
                self.implementation,
                &mut buf,
            ))?;
        }
        Ok(buf as *mut u8)
```
Can we reduce the scope of the unsafe block here to only include the code that must be in an unsafe block, or is there some reason the buff needs to be declared and returned within the unsafe block? This doesn't seem very functionally different from declaring the entire function unsafe except that the caller doesn't have to put it in an unsafe block.